<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateIt</class>
 <widget class="QMainWindow" name="CreateIt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreateIt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="central_frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="red_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(129, 9, 9);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>180</width>
        <height>209</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="red_Layout">
       <item>
        <widget class="QLabel" name="welcome_label">
         <property name="font">
          <font>
           <family>Rosemary</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Welcome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="by">
         <property name="font">
          <font>
           <family>Rosemary</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>To</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="createit">
         <property name="font">
          <font>
           <family>Rosemary</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>CreateIt</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="code_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>266</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>This Code is Developed &amp; Maintained by</string>
      </property>
     </widget>
     <widget class="QLabel" name="sanat_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>175</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SANAT KUMAR MISHRA | PSIT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="gray_frame">
     <property name="geometry">
      <rect>
       <x>299</x>
       <y>-1</y>
       <width>450</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(214, 214, 214);

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="enter_detail_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter Details:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_btn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>410</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Consolas&quot;;
background-color: rgb(170, 170, 170);
</string>
      </property>
      <property name="text">
       <string>CREATE</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>358</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="folder_layout">
       <item>
        <widget class="QLabel" name="select_path_label">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select Folder Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="u_folder_layout">
         <item row="0" column="0">
          <widget class="QLineEdit" name="input_path">
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>S E L E C T   P A T H  </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="browse">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 170);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>254</width>
        <height>98</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="programming_layout">
       <item>
        <widget class="QLabel" name="programming_label">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select Programming Language:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="lang_layout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="cplus">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>C++</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="java">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Java</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="python">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Python</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>328</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="time_delta_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="time_label">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enter Time Delta:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="time_delta_input">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string>0 </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>304</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="file_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="file_label">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enter File Name :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="add_file_btn">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 170);</string>
         </property>
         <property name="text">
          <string>Add Files Names</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
